{"uuid":"0077df05-8ed8-4e42-99c8-fb4951628ca6","historyId":null,"status":"broken","statusDetails":{"message":"Oops...we found an error preparing this test file:\n\n  > cypress/e2e/percy/visual-testing-percy.cy.js\n\n","trace":"\n\nError: Webpack Compilation Error\nModule not found: Error: Can't resolve '../fixtures/pages.json' in '/Users/gutierrezme/Documents/QA/qa-challenge-manu/cypress/e2e/percy'\n    at handle (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/@cypress/webpack-preprocessor/dist/index.js:212:23)\n    at finalCallback (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:441:32)\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:505:17\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/HookWebpackError.js:68:3\n    at Hook.eval [as callAsync] (eval at create (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/Hook.js:18:14)\n    at Cache.storeBuildDependencies (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Cache.js:122:37)\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:501:19\n    at Hook.eval [as callAsync] (eval at create (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/Hook.js:18:14)\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:498:23\n    at Compiler.emitRecords (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:919:5)\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:490:11\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:885:14\n    at Hook.eval [as callAsync] (eval at create (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:6:1)\n    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/tapable/lib/Hook.js:18:14)\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:882:27\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/neo-async/async.js:2818:7\n    at done (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/neo-async/async.js:3522:9)\n    at alreadyWritten (/Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:714:8)\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/webpack/lib/Compiler.js:802:19\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/node_modules/graceful-fs/graceful-fs.js:123:16\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:123:16\n    at /Users/gutierrezme/Library/Caches/Cypress/13.17.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/graceful-fs/graceful-fs.js:123:16\n    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read/context:68:3)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests."},"stage":"finished","attachments":[],"parameters":[],"labels":[{"name":"suite","value":"visual-testing-percy.cy.js"}],"links":[],"start":null,"name":"Oops...we found an error preparing this test file: visual-testing-percy.cy.js","fullName":"Oops...we found an error preparing this test file: visual-testing-percy.cy.js","stop":null}